udata_ods/harvesters.py

```diff
@@ -125,24 +129,43 @@ def process(self, item):
         if 'geo' in ods_dataset['features']:
             self.process_resources(dataset, ods_dataset, ('geojson', 'shp'))
 
+        if 'alternative_exports' in ods_dataset:
+            self.process_alternative_exports(dataset, ods_dataset)
+
         dataset.extras["ods:url"] = self.explore_url(dataset_id)
         if "references" in ods_metadata:
             dataset.extras["ods:references"] = ods_metadata["references"]
         dataset.extras["ods:has_records"] = ods_dataset["has_records"]
 
         return dataset
 
+    def process_alternative_exports(self, dataset, data):
+        dataset_id = data['datasetid']
+        modified_at = data['metas']['modified']
+        for export in data['alternative_exports']:
+            data = {
+                'title': export.get('title', 'No title'),
+            }
+            if 'description' in export:
+                data['description'] = export['description']
+            if 'mimetype' in export:
+                data['mime'] = export['mimetype']
+            data['url'] = self.alternative_export_url(dataset_id, export['id'])
+            resource = Resource(**data)
+            resource.modified = modified_at
+            dataset.resources.append(resource)
```

Comment:
I think we need to do something to avoid exports to be appended at each run. Rely on `url` or `export_id` to check if the resource already exists or not. Maybe store the export id as `ods:export_id` extra

Reply:
Isn't the dataset fully recreated at each run? Because there does not seem to be duplicates (cf new test for idempotence).

Reply:
No, the harvest engine fetch existing dataset if there is one. But from what I see, the ODS harvester seems to replace resources  at each run (ie. `resource.id` should change each time :/)